lib/analytics_data.rb

```diff
@@ -45,4 +45,11 @@ def columns
       "ga:dimension80" => lambda { |_| nil }, # Placeholder: is page a best bet?
     }
   end
+
```

Comment:
Maybe worth naming this "for_google_analytics" or adding a comment explaining it. 

Otherwise it looks unnecessary at a glance since you're running it through the CSV module.